src/styles/styles.js

```diff
@@ -1982,6 +1982,17 @@ const styles = {
         backgroundColor: colors.white,
     },
 
+    radioButtonContainer: {
+        backgroundColor: themeColors.componentBG,
+        borderRadius: 10,
+        height: 20,
+        width: 20,
+        borderColor: themeColors.border,
```

Comment:
We've gotten feedback before that the border color for these feels way too light. Maybe that's a separate problem, but I could see bumping up the gray one step to using themeColors.icon which would make it a bit darker. Could I see a before/after on that so we can take a look? Thanks!

Reply:
Before:

![image](https://user-images.githubusercontent.com/2463975/157215918-2498ebbc-5a1e-485e-86f6-744138d8659f.png)


After:

![image](https://user-images.githubusercontent.com/2463975/157215851-802d725b-f068-4260-9308-c15eb1f7ede1.png)